include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -23,6 +23,7 @@ typedef struct _oe_sgx_enclave_image_info_t
     uint64_t reloc_size;
     uint64_t heap_rva; /* heap size is in header.sizesettings */
     uint64_t enclave_size;
+    uint64_t entry_rva;
```

Comment:
Although the latest EEID design proposal does not rely on `entry_rva` in _oe_sgx_enclave_image_info_t. I stil think this change is valuable for future EDMM work.